content/glossary/sql/case.md

```diff
@@ -0,0 +1,41 @@
+<!--
```

Comment:
remove "sql" from tags and subjects, title from markdown

Reply:
👍🏼